apps/dev/dbschema/default.esdl

```diff
@@ -0,0 +1,71 @@
+module default {
+    type User {
+        property name -> str;
+        required property email -> str {
+            constraint exclusive;
+        }
+        property emailVerified -> datetime;
+        property image -> str;
+        multi link accounts := .<user[is Account];
+        multi link sessions := .<user[is Session]; 
+        property createdAt -> datetime {
+            default := datetime_current();
+        };
+    }
+
+    type Account {
+       required property userId -> str;
```

Comment:
The `userId` field here and on `Session` seem redundant, since there's already a link to `User`. If `userId` is needed in a query, it should be a computed property instead.